reporting-bugs/readme.md

```diff
@@ -0,0 +1,263 @@
+# Asking Questions 101
+
+## Prep
+
+- [ ] Read this article about [Mental Models](https://jamesclear.com/mental-models)
+- [ ] Review the Asking Questions workshop and the CYF Guide to Asking Questions on the curriculum.
+
+Today we're going to build on our formal language of developer questions. We began with this basic format:
+
+1. What I _did_
+1. What I _expected_
+1. What actually _happened_
+
+This format helps to find the discrepancies between expectations and reality. (This is the gap in our understanding.)
+
+It's also an efficient way to share our current understanding with others. Now we're going to use this to write a really good bug report. A well-written bug report can help developers identify, reproduce, and fix issues more efficiently. We will follow this basic format:
+
+1. Steps to Reproduce (What I _did_)
+1. Expected Behavior (What I _expected_)
+1. Actual Behavior (What actually _happened_)
+1. Environment Details 🆕
+
+> Clarity is our goal
+
+## Exchanging incomplete information sets 30m
+
+Over and over, especially during The Launch, we see trainees writing panicked, incoherent questions with no context, pasting error messages they clearly have not read, and not even sharing the link to the code they're working on.
+
+Outside our community, developers will ignore or delete your posts if you do this. At home with us, well-meaning mentors often try to help you, but they _can't_ because you haven't given them the information they need. It feels so frustrating, and people can get really upset. We want to help you avoid this as you prepare to work in a professional environment.
```

Comment:
_"developers will ignore your posts if you do this"_ - we could couple this more explicitly to employability:

- Useful questions -> colleagues enjoy answering -> get solutions promptly -> make lots of progress -> good performance
- Not useful questions -> colleagues avoid answering -> don't get solutions -> take much longer -> performance appears poor